src/app/services/schema/app-schema.service.ts

```diff
@@ -239,7 +239,7 @@ export class AppSchemaService {
         const fieldSchema = _.find(schema?.questions, { variable: key });
 
         if (fieldSchema?.schema?.null) {
-          return null;
+          return;
```

Comment:
in this case we need to return `null`.
**return;** -> will end up with **undefined** but we need `null` exactly I assume, because field is **nullable** and we should be able to submit **null**, but not sure about _undefined_.

Reply:
This is within `forEach`. It doesn't return anywhere.